src/fidesops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -226,10 +230,22 @@ def privacy_request_csv_download(privacy_request_query: Query) -> StreamingRespo
             "Request status",
             "Reviewer",
             "Time approved/denied",
+            "Denial reason",
         ]
     )
+    privacy_request_ids: List[str] = [r.id for r in privacy_request_query]
+    audit_log_query = db.query(AuditLog).filter(
+        AuditLog.action == AuditLogAction.denied.value,
+        AuditLog.privacy_request_id.in_(privacy_request_ids),
+    )
+    denial_audit_logs = {r.privacy_request_id: r.message for r in audit_log_query}
```

Comment:
can we get more return types added in this method, overall?

Reply:
yes, I added type annotations to line 237 and 241